src/Http/WebUtilities/src/ReasonPhrases.cs

```diff
@@ -88,6 +182,14 @@ public static class ReasonPhrases
     /// <returns>The reason phrase, or <see cref="string.Empty"/> if the status code is unknown.</returns>
     public static string GetReasonPhrase(int statusCode)
     {
-        return Phrases.TryGetValue(statusCode, out var phrase) ? phrase : string.Empty;
+        if (statusCode >= 100 && statusCode < 600)
```

Comment:
```suggestion
        if ((uint)(statusCode - 100) < (600 - 100))
```
This safes a comparison and a branch.